src/Kunstmaan/SeoBundle/Controller/Admin/SettingsController.php

```diff
@@ -68,6 +51,11 @@ public function robotsSettingsAction(Request $request)
             }
         }
 
+        if ($robot->getRobotsTxt() === $default) {
```

Comment:
you should show the warning only when the value is null, and if the value is null, the default should be used in the "front"